keyboards/kiwikeebs/macro-v2/readme.md

```diff
@@ -0,0 +1,19 @@
+# KiwiKeebs MacroBoard V1.0
+
+![KiwiKeebs](https://i.imgur.com/h5i9jBl.png)
+
+A 6 Key PCB with Rotary Encoder Knob + Button made from solid CNC aluminium.
+
+* Keyboard Maintainer: [AKiwi92](https://github.com/akiwi92)
+* Hardware Supported: KiwiKeebs MacroBoard V1.0
+* Hardware Availability: [reddit.com](https://www.reddit.com/r/mechmarket/comments/ibijrd/ic_kiwikeebs_6key_macroboard_rotary_encoder_diy/)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to jump to the bootloader?

Reply:
Added instructions on how to put into DFU mode